src/configuring-wire-formats.md

```diff
@@ -18,6 +18,7 @@ Option|Default|Description
 `maxInactivityDuration`|`30000`|The maximum [inactivity](activemq-inactivitymonitor) duration (before which the socket is considered dead) in milliseconds. On some platforms it can take a long time for a socket to die. Therefore allow the broker to kill connections when they have been inactive for the configured period of time. Used by some transports to enable a keep alive heart beat feature. Inactivity monitoring is disabled when set to a value `<= 0`.
 `maxInactivityDurationInitalDelay`|`10000`|The initial delay before starting [inactivity](activemq-inactivitymonitor) checks. Yes, the word `'Inital'` is supposed to be misspelled like that.
 `maxFrameSize`|`MAX_LONG`|Maximum allowed frame size. Can help help prevent OOM DOS attacks.
```

Comment:
I would update this description to mention this setting applies to both client and server side but that the minimum value is chosen so a client can't pick a larger value than the server, etc. Also you could mention here that as of 5.16.4 the size is checked client side by default unless maxFrameSizeEnabled is false (and obviously your other update describes what the flag does)